Formula/m/mbpoll.rb

```diff
@@ -0,0 +1,29 @@
+class Mbpoll < Formula
+  desc "Command-line utility to communicate with ModBus slave (RTU or TCP)"
+  homepage "https://epsilonrt.fr"
+  url "https://github.com/epsilonrt/mbpoll/archive/refs/tags/v1.5.2.tar.gz"
+  sha256 "7d960cd4459b5f7c2412abc51aba93a20b6114fd75d1de412b1e540cfb63bfec"
+  license "GPL-3.0-only"
+  head "https://github.com/epsilonrt/mbpoll.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libmodbus" => ["3.1.4"]
```

Comment:
This doesn't do anything
```suggestion
  depends_on "libmodbus"
```

Reply:
But the [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook) documentation says:
`Some Requirements can also take a string or symbol specifying their minimum version that the formula depends on.`

Reply:
Yes, those requirements are `:xcode` and nothing else anymore I believe.

Reply:
yeah, better to remove it, because when you do `depends_on "libmodbus"`, you always use latest libmodbus, so there is no reason to check the version at all (currently the version is 3.1.10)